components/builder-depot/src/metrics.rs

```diff
@@ -33,10 +33,10 @@ impl metrics::Metric for Counter {
     fn id(&self) -> Cow<'static, str> {
         match *self {
             Counter::SearchPackages => "search-packages".into(),
-            Counter::UploadRequests => "upload-packages".into(),
-            Counter::SingleUploadRequests => "upload-packages".into(),
-            Counter::MultipartUploadRequests => "upload-packages".into(),
-            Counter::DownloadRequests => "download-packages".into(),
+            Counter::UploadRequests => "upload-requests".into(),
```

Comment:
I'm curious why you renamed to requests instead of packages.. 'upload-packages' seems more specific and consistent with the other metrics eg 'search-packages'..

Reply:
I can change it back, i just figured upload-requests was more accurate to what was happening, the metric there only counts if the request has been made, not if its completed. Im ambivalent though, i can switch it back if you'd prefer.